frappe/installer.py

```diff
@@ -267,6 +266,10 @@ def install_app(name, verbose=False, set_as_patched=True, force=False):
 			required_app = parse_app_name(app)
 			install_app(required_app, verbose=verbose, force=force)
 
+	telemetry_enabled = frappe.get_hooks(app_name=name, default={}).get("enable_telemetry", False)
+	if telemetry_enabled:
+		frappe.get_single_doc("Telemetry Settings").enable_telemetry(app)
+
```

Comment:
This should also run on say `after_migrate`. An app may disable telemetry in the beginning and later enable it.